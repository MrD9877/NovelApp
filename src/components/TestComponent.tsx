import React from "react";

export default function TestComponent() {
  return <h1>Test</h1>;
}
