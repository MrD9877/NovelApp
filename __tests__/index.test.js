import "@testing-library/jest-dom";
import { render, cleanup } from "@testing-library/react";
import TestComponent from "../src/components/TestComponent";

describe("Home", () => {
  afterEach(cleanup);
  it("renders a heading", () => {
    const { asFragment } = render(<TestComponent />);

    expect(asFragment(<TestComponent />)).toMatchSnapshot();
  });
});
