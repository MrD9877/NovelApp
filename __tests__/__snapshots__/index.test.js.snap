// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<DocumentFragment>
  <h1>
    Test
  </h1>
</DocumentFragment>
`;
